status.inc.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Get approot.inc.php
+ * Move to a function for allowing a better testing
+ * 
+ * @param string $sAppRootFilename
+ * @throws \Exception
+ */
+function StatusGetAppRoot($sAppRootFilename = 'approot.inc.php') 
+{
+        $sAppRootFile = dirname(__FILE__).'/'.$sAppRootFilename;
+
+        /*
+        * Check that the approot file exists and has the appropriate access rights
+        */
+        if (!file_exists($sAppRootFile) || !is_readable($sAppRootFile)) 
+        {
+            throw new \Exception($sAppRootFile . ' is not readable');
+        }
+        require_once($sAppRootFile);
+}
+
+/**
+ * Check iTop's config File existence and readability
+ * Move to a function for allowing a better testing
+ * 
+ * @param string $sConfigFilename
+ * @throws \Exception
+ */
+function StatusCheckConfigFile($sConfigFilename = 'config-itop.php')
+{
+        StatusGetAppRoot();
+        
+        $sConfigFile = APPCONF.ITOP_DEFAULT_ENV.'/'.$sConfigFilename;
+
+        /**
+         * Check that the configuration file exists and has the appropriate access rights
+         */  
+        if (!file_exists($sConfigFile) || !is_readable($sConfigFile)) 
+        {
+                throw new \Exception($sConfigFile . ' is not readable');
```

Comment:
Same as the previous comment.
```suggestion
                throw new \Exception($sConfigFilename . ' is not readable');
```